### PR TITLE
Ep13016 convert graphics app dashboard to ng6

### PR DESCRIPTION
**Pull request by seishin4real**
+ acount: hasFeatureFlag()
+ dashboard: openContentIntervalSettingsDialog()
  (moved content interval settings dialog to dashboard to avoid direct api calls from graphics dashboard.)
+ npm publish scripts

new functionality in this branch depends on equiv. branch in `dashboard` project

### CODE REVIEW

File: dist/sdk.js [hunk 1]
```diff
@@ -686,6 +686,24 @@
...
 
+        /**
+         * Returns flag status on account
+         * @param {string} flagName
+         * @param {function} [onSuccess]
+         * @param {function} [onError]
+         * @returns {number} callId
+         */
+        this.hasFeatureFlag = function (flagName, onSuccess, onError) {
+            return this.method({
```
Comment (JerryDoubleU) [inline dist/sdk.js:698]: `this.validate` validation missing - see this.deleteTheme method for example

File: dist/sdk.js [hunk 2]
```diff
@@ -1182,6 +1200,29 @@
             });
         };
 
+        /**
+        * // todo
```
Comment (JerryDoubleU) [inline dist/sdk.js]: comment missing

File: dist/sdk.js [hunk 3]
```diff
@@ -1182,6 +1200,29 @@
...
+        *
+        * @param {string} appName
+        * @param {string} level - Enplug | Account | Venue
+        * @param {string} levelId
+        * @param {function} onSuccess
+        * @param {function} onError
+        * @returns {number} callId
+        */
+        this.openContentIntervalSettingsDialog = function (appName, level, levelId, onSuccess, onError) {
+            return this.method({
```
Comment (JerryDoubleU) [inline dist/sdk.js:1219]: missing parameter validation

File: src/account.js [hunk 4]
```diff
@@ -400,6 +400,24 @@
...
 
+        /**
+         * Returns flag status on account
+         * @param {string} flagName
+         * @param {function} [onSuccess]
+         * @param {function} [onError]
+         * @returns {number} callId
+         */
+        this.hasFeatureFlag = function (flagName, onSuccess, onError) {
+            return this.method({
```
Comment (JerryDoubleU) [inline src/account.js:412]: missing parameter validation

File: src/dashboard.js [hunk 5]
```diff
@@ -478,6 +478,29 @@
...
+        *
+        * @param {string} appName
+        * @param {string} level - Enplug | Account | Venue
+        * @param {string} levelId
+        * @param {function} onSuccess
+        * @param {function} onError
+        * @returns {number} callId
+        */
+        this.openContentIntervalSettingsDialog = function (appName, level, levelId, onSuccess, onError) {
+            return this.method({
```
Comment (JerryDoubleU) [inline src/dashboard.js:496]: missing parameter validation

File: src/dashboard.js [hunk 6]
```diff
@@ -478,6 +478,29 @@
             });
         };
 
+        /**
+        * opens content interval settings dialog for ad-sheduler app
```
Comment (JerryDoubleU) [inline src/dashboard.js]: typo in comment `sheduler` -> `scheduler'

**Review by JerryDoubleU — CHANGES_REQUESTED** [review]

**Review by justynaw — APPROVED** [review]

File: dist/sdk.js (hunk 1 above)
Comment (seishin4real) [reply dist/sdk.js:698]: good point. fixed

File: dist/sdk.js (hunk 2 above)
Comment (seishin4real) [reply dist/sdk.js]: fixed

File: src/account.js (hunk 4 above)
Comment (seishin4real) [reply src/account.js:412]: good point. fixed

File: src/dashboard.js (hunk 5 above)
Comment (seishin4real) [reply src/dashboard.js:496]: good point. fixed

File: src/dashboard.js (hunk 6 above)
Comment (seishin4real) [reply src/dashboard.js]: keen eye :) fixed

File: dist/sdk.js (hunk 3 above)
Comment (seishin4real) [reply dist/sdk.js:1219]: good point, fixed

**Review by JerryDoubleU — APPROVED** [review]

**Pull request merged**